nestkernel/recording_device.cpp

```diff
@@ -290,6 +295,14 @@ nest::RecordingDevice::State_::get( DictionaryDatum& d,
       dict, names::weights, std::vector< double >( event_weights_ ) );
   }
 
+  if ( p.withreceivergid_ )
```

Comment:
i was wondering, what's the use case of not recording the receiver gid? don't you always need at least the tuple (sender, receiver) to make sense out of the recorded data? or is this just signalling that this is a `weight_recorder`? in that case, maybe you could use a more explicit variable name?